tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -372,13 +373,13 @@ def _ValidateLinkStatesMatch(self, universe, source_field, target_field):
     source_states = universe.GetStatesByField(source_field)
     target_states = universe.GetStatesByField(target_field)
     if source_states != target_states:
-      print('State mismatch in link from {0} to {1}'\
+      print('State mismatch in link from {0} to {1}'
             .format(source_field, target_field))
       return False
     return True
 
 
-  def IsValidEntityInstance(self, universe = None, entity_instances = None):
+  def IsValidEntityInstance(self, universe=None, entity_instances=None):
```

Comment:
removing the spaces, I'm pretty sure, is incorrect.

Reply:
I have this error from the linter:
```
validate/entity_instance.py:382:43: C0326: No space allowed around keyword argument assignment
  def IsValidEntityInstance(self, universe = None, entity_instances = None):
                                           ^ (bad-whitespace)
validate/entity_instance.py:382:68: C0326: No space allowed around keyword argument assignment
  def IsValidEntityInstance(self, universe = None, entity_instances = None):
                                                                    ^ (bad-whitespace)
```